app/config/config.yml

```diff
@@ -83,7 +83,7 @@ doctrine:
                 user:     "%database_user%"
                 password: "%database_password%"
                 server_version: 5.1
```

Comment:
MySQL 5.1 doesn't support utf8mb4. Your MySQL server must have a version 5.5.3 or higher (including all 10.x versions of MariaDB).

Reply:
trolololol
> Always wrap the server version number with quotes to parse it as a string instead of a float number. Otherwise, the floating-point representation issues can make your version be considered a different number (e.g. 5.6 will be rounded as 5.5999999999999996447286321199499070644378662109375).

I remove this configuration key because we can't specifcy which version the merchant is running. Nice catch!

Reply:
:+1: